lib/srv/db/server_test.go

```diff
@@ -236,6 +237,16 @@ func TestDatabaseServerAutoDisconnect(t *testing.T) {
 	require.NoError(t, pgConn.Close(ctx))
 }
 
+func advanceInSteps(clock clockwork.FakeClock, total time.Duration) {
```

Comment:
This one could do with a comment, as generally this isn't something we need to be worried about.